flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/proxy/KinesisProxy.java

```diff
@@ -313,6 +313,9 @@ public GetRecordsResult getRecords(String shardIterator, int maxRecordsToGet)
                 } else {
                     throw ex;
                 }
+            } catch (RuntimeException ex) {
+                LOG.error(ex.getClass().getName() + ": " + ex.getMessage());
```

Comment:
Why not `LOG.error(ex)`, or something with a meaningful message `LOG.error("Encountered non-recoverable error while invoking getRecords.", ex);`